unreleased_history/bug_fixes/opt_seek.md

```diff
@@ -0,0 +1 @@
+During seek, the iterator seeks every file on L0. In async_io, it submit the requests to seek on every file on L0 asynchronously using RocksDB FilePrefetchBuffer. However, FilePrefetchBuffer does alignment and reads extra bytes then needed that can increase the throughput. Added a fix to avoid that alignment when direct_io is disabled.
```

Comment:
"Every file on L0, and every non-zero level"

Reply:
I think we can omit the detail of seeking every file in L0 and other levels. Just say that when reading a block for seeking a target key in a file and not doing readahead, the iterator was aligning the read on a page boundary and reading more than necessary. This increases the storage read bandwidth usage.